AnnoyancesFilter/sections/antiadblock.txt

```diff
@@ -207,6 +207,7 @@ zkillboard.com###adsensetop
 @@||pagead2.googlesyndication.com/pagead/js/adsbygoogle.js$domain=zkillboard.com
 filmix.cc,filmix.gold,filmix.vip,filmix.fun,filmix.cool,filmix.nl,filmix.info,filmix.me,filmix.co##.social-counters
 nekopoi.care##.mobileadstop
+dist-tutor.info$$script[tag-content="addblock"][max-length="850"]
```

Comment:
is it required?

Reply:
To be honest, this list of rules is enough for that site. So I deleted rule.